### ISSUE TITLE
Allow dumping scrollback as transcript

### ISSUE THREAD

**Issue by GoogleCodeExporter** [garglk/garglk#145]
```
Sometimes, I forget to record a transcript.

Since Gargoyle already has a scrollback, it would be nice to be able to dump it 
to a file.
```

Original issue reported on code.google.com by `saltyho...@gmail.com` on 26 Mar 2011 at 8:08

**Comment by cspiegel** [comment]
Now, after 13 years, I've got a pull request that implements this, in #804. Control-shift-S (command-shift-s on Mac) should allow the scrollback buffer to be saved.  I doubt anybody is monitoring this migrated issue, but feel free to leave comments on the pull request if there are any problems.  Otherwise, I'll merge it and any issues can be fixed up down the road, if they're discovered.

**Comment by cspiegel** [comment]
Note also that this is still _not_ a suitable replacement for a proper transcript, since, as noted, the scrollback can be cleared (and it's also limited in size). But it at least gives you a fighting chance of saving some stuff if you forget to start transcripting.

Related to this, Bocfel (the Z-machine interpreter) does support automatic transcripting. The [man page](https://cspiegel.github.io/bocfel/bocfel.html) documents it (look for "persistent", since the feature is called "persistent transcripting").  When turned on, games will automatically store a transcript in memory (and also in save files), and it's always available to save to disk.

**Comment by cspiegel** [comment]
Fixed in #804
